### DISCUSSION TITLE
Alert window enhacement

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#397] [Ideas]
![image](https://github.com/foldynl/QLog/assets/61935185/3f91e74c-7b2c-4518-a65a-b13681ebaac7)
I don't know if it would be possible for the alert window to be included in the layout so that when you start the software it stays in the desired position? As you can see in the picture.

by the way, where I can help with the translation of the program?

**Comment by foldynl** [comment]
to be honest, I have to think about why I didn't make it part of the main layout. Probably due to the fact that it was designed to be only a temporary view. But I really don't know why.

Related to the translation, I have added a procedure for that to the [wiki](https://github.com/foldynl/QLog/wiki/Multi-language-Support#how-to-create-a-translation-file).
